src/app/components/cds/media-cards/Card.js

```diff
@@ -78,51 +44,14 @@ const Card = ({
     >
       <MaybeLink to={cardUrl}>
         <div className={styles.cardContent}>
-          <div className={styles.cardLeft}>
-            <h6 className={`typography-button ${styles.cardTitle}`}>{title}</h6>
-            { description ?
-              <p className="typography-body2">
-                <span className={`description-text ${styles.cardDescription} ${isCollapsed ? styles.cardDescriptionCollapse : ''}`} ref={descriptionRef}>
-                  {description}
-                </span>
-                { factCheckUrl ?
-                  <span className={styles.factCheckLink}>
-                    <FactCheckIcon />
-                    {' '}
```

Comment:
I wanna get rid of stuff like this, we shouldn't be spacing with text!

Reply:
agreed...css ftw